lib/wallaby/experimental/selenium.ex

```diff
@@ -0,0 +1,158 @@
+defmodule Wallaby.Experimental.Selenium do
+  @moduledoc false
+
+  @behaviour Wallaby.Driver
+
+  alias Wallaby.{Element, Session}
+  alias Wallaby.Experimental.Selenium.WebdriverClient
+
+  @type start_session_opts ::
+    {:remote_url, String.t} |
+    {:capabilities, map} |
+    {:create_session_fn, ((String.t, map) -> {:ok, %{}})}
+
+  @spec start_session([start_session_opts]) :: {:ok, Session.t}
+  def start_session(opts \\ []) do
+    base_url = Keyword.get(opts, :remote_url, "http://localhost:4444/wd/hub/")
```

Comment:
I suspect this option will need to change to something like `:selenium_remote_url` once we add chromedriver support as well.